apps/src/lib/client/rpc.rs

```diff
@@ -1839,13 +1839,13 @@ pub async fn query_conversions<C: namada::ledger::queries::Client + Sync>(
     // Track whether any non-sentinel conversions are found
     let mut conversions_found = false;
     for ((addr, _), epoch, conv, _) in conv_state.assets.values() {
-        let amt: masp_primitives::transaction::components::Amount =
+        let amt: masp_primitives::transaction::components::I32Sum =
             conv.clone().into();
         // If the user has specified any targets, then meet them
         // If we have a sentinel conversion, then skip printing
         if matches!(&target_token, Some(target) if target != addr)
             || matches!(&args.epoch, Some(target) if target != epoch)
-            || amt == masp_primitives::transaction::components::Amount::zero()
+            || amt == masp_primitives::transaction::components::ValueSum::zero()
```

Comment:
nit: I think it would be nice to have an `is_zero` instance method, so we could call `amt.is_zero()`. That way we don't have two different types for `amt` within a couple of lines (`I32Sum` and `ValueSum`). It would be nice to read this code without needing to go check a type alias.

Reply:
Good idea.